### ISSUE TITLE
"Import Site" reveal private IP and port service

### ISSUE THREAD

**Issue by 410g0n3** [gophish/gophish#3016]
When you click on the "Import site" button and the application performs a clone of the web page, it reflects on the backend of the web server the private address and the port on which the service runs.
Can this information be removed from the headers when performing the import site?

![imagen](https://github.com/gophish/gophish/assets/76210079/5b5bbdfc-d6aa-4003-a84d-96db2bd4d7d5)

**Comment by Miesvanderlippe** [comment]
Requesting a website will always reveal your IP address to the server you're requesting it from. That's how the internet works. 

You probably do not have the required legal clearance to do phishing if you're asking this question. I suggest this issue to be closed without further answers.

**Comment by 410g0n3** [comment]
@Miesvanderlippe 
Yes, that's how the Internet works when it comes to public IPs. As I said what is revealed is the private address, in the attached image it is a class A IP (10.x.x.x). Which is being reflected in the logs of the website you are "importing site" to.
Anyway, thanks for your interest in this issue.
